components/gc-featured-link/gc-featured-link-en.html

```diff
@@ -44,23 +44,7 @@ <h2 class="wb-inv">Featured link</h2>
 	</div>
 </div>
 
-<h3>Code</h3>
-<pre><code>
-&lt;div class="provisional gc-featured-link"&gt;
-	&lt;div class="container"&gt;
-		&lt;div class="row"&gt;
-			&lt;div class="col-xs-12"&gt;
-				&lt;h2 class="wb-inv"&gt;Featured link&lt;/h2&gt;
-				&lt;p&gt;&lt;a class="stretched-link" href="#"&gt;Featured link [optional]&lt;/a&gt;&lt;/p&gt;
-			&lt;/div&gt;
-		&lt;/div&gt;
-	&lt;/div&gt;
-&lt;/div&gt;
-</code></pre>
-
-<h3>Home page pattern (old vs new)</h3>
-
-<h4>Old pattern using helpers</h4>
+<h3>Featured link - Home page pattern</h3>
```

Comment:
This example should be deeper in the page instead of being the first. That design pattern that leverage utilities/helpers CSS class is now a deprecated in favour of the new provisional component. 

Instead here, add a working demo as per how this new feature should be coded.